src/util.c

```diff
@@ -40,6 +40,7 @@ void luv_stack_dump(lua_State* L, const char* name) {
 }
 
 static int luv_error(lua_State* L, int status) {
+  assert(status != 0);
```

Comment:
assert(status < 0) ?

Reply:
That'd probably be better. Would have the potential to catch Libuv bugs where Windows error codes aren't being translated properly.